setup.py

```diff
@@ -178,15 +178,14 @@ def run(self):
     packages=find_packages("Lib"),
     entry_points={
         "console_scripts": [
-            "glyphs2ufo = glyphsLib.__main__:main"
+            "glyphs-converter = glyphsLib.cli:main",
```

Comment:
can't we just remove this console script entry point altogether and simply use `python -m glyphsLib [glyphs2ufo|ufo2glyphs]`? It's not much more typing after all.

Reply:
Hm. Somewhat hesitant, but okay.

Reply:
i'm okay if you want to have two separate console scripts `glyphs2ufo` and `ufo2glyphs`, but `glyphs-coverter [glyphs2ufo|ufo2glyphs]` seems too much typing for nothing, especially since `python -m glyphsLib [glyphs2ufo|ufo2glyphs]` would do the same (and I want to keep the latter).

maybe the original method you were using of parsing only the first argument in the top-level glyphsLib main, and then pass through all the following unparsed arguments to the respective sub-main functions (glyphs2ufo and ufo2glyphs) which do their own argparsing, works better than the argparse sub-parsers approach?
Then we could keep both a generic `python -m glyphsLib` invocation method (with `glyphs2ufo` and `ufo2glyphs` as subcommands of the latter), and two specific console scripts, respectively `glyphs2ufo` and `ufo2glyphs` for those who prefer less typing

Reply:
Sigh, yeah I'm not entirely happy with the current approach either... okay, so I'll revert back to the previous way.

Reply:
Wait. I got a better idea.
```
def _glyphs2ufo_entry_point():
    args = sys.argv[1:]
    args.insert(0, "glyphs2ufo")
    return main(args)
[...]
```
And in setup.py:
```
        "console_scripts": [
            "ufo2glyphs = glyphsLib.cli:_ufo2glyphs_entry_point",
            "glyphs2ufo = glyphsLib.cli:_glyphs2ufo_entry_point",
        ],
```